students/DrewSmith/lessons/lesson03/assignment/create_customers.py

```diff
@@ -0,0 +1,44 @@
+"""
+    Manage database environment with Peewee ORM, sqlite and Python
+"""
+
+import logging
+import sqlite3
+from sqlite3 import Error
+import peewee as pw
+
+
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+DATABASE = None
```

Comment:
Database shouldn't be a constant (it is a database represented as a python object, which is not constant).  I like that you have come up with a method to have two different databases. Much better than overwriting the database when you run your tests. :) You can look at my example to see how to do it with a little less code.